File: 34-find-first-and-last-position-of-element-in-sorted-array/34-find-first-and-last-position-of-element-in-sorted-array.cs
public class Solution {
    public int[] SearchRange(int[] nums, int target) {
        
        var ind = new int[2]{-1,-1};
        
        ind[0] = Array.IndexOf(nums,target);
        
        ind[1] = Array.LastIndexOf(nums,target);
        
        return ind;
                          
    }
}

File: 34-find-first-and-last-position-of-element-in-sorted-array/34-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int> v(2,-1);
        auto fun = equal_range(nums.begin(),nums.end(),target);
        if(fun.first==fun.second){
            return v;
        }
            v[0]=fun.first-nums.begin();
            v[1]=fun.second-1-nums.begin();
            return v;
    }
};